addon/webextension/background/selectorLoader.js

```diff
@@ -0,0 +1,83 @@
+/* globals browser, catcher */
+window.selectorLoader = (function () {
+  const exports = {};
+  // These modules are loaded in order, and some need to be listed before others
+  // due to dependencies:
+  const scripts = [
+    "catcher.js",
+    "background/selectorLoader.js",
+    "selector/callBackground.js",
+    "clipboard.js",
+    "makeUuid.js",
+    "build/shot.js",
+    "randomString.js",
+    "domainFromUrl.js",
+    "build/inlineSelectionCss.js",
+    "selector/documentMetadata.js",
+    "selector/util.js",
+    "selector/ui.js",
+    "selector/shooter.js",
+    "selector/uicontrol.js"
+  ];
+
+  exports.unloadIfLoaded = function () {
+    return browser.tabs.executeScript({
+      code: "window.selectorLoader && window.selectorLoader.unloadModules()"
+    }).then(result => {
+      return result && result.toString() === "true";
+    });
+  };
+
+  exports.loadModules = function () {
+    let lastPromise = Promise.resolve(null);
+    scripts.forEach((file) => {
+      lastPromise = lastPromise.then(() => {
```

Comment:
The [docs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/executeScript) don't seem to mention the ability to pass in a list.  I think the initial version of this used `chrome.tabs.executeScript()` which didn't allow a list when I was experimenting with it.